<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TempInvalidPredecessorsSelector</class>
 <widget class="QDialog" name="TempInvalidPredecessorsSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Действующие ЭЛН по данным СФР</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="1" column="1">
    <widget class="CDateEdit" name="edtBegExternalPeriod">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="CInDocTableView" name="tblPredecessors"/>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="chkExternalPeriod">
     <property name="text">
      <string>Внешний период нетрудоспособности с</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="CDateEdit" name="edtEndExternalPeriod">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="lblEndExternalPeriod">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>по</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tblPredecessors</tabstop>
  <tabstop>chkExternalPeriod</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TempInvalidPredecessorsSelector</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>4</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TempInvalidPredecessorsSelector</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkExternalPeriod</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtBegExternalPeriod</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkExternalPeriod</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblEndExternalPeriod</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkExternalPeriod</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtEndExternalPeriod</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
